Failing on glide install

Getting these errors when trying to run the glide install command
```
$ glide install 
[ERROR] Error scanning k8s.io/kube-openapi/pkg/util/proto: cannot find package "." in:
        /Users/<USER>/.glide/cache/src/https-k8s.io-kube-openapi/pkg/util/proto
[ERROR] Error scanning github.com/docker/docker/pkg/homedir: cannot find package "." in:
        /Users/rnowacoski/.glide/cache/src/https-github.com-docker-docker/pkg/homedi
```

Able to install with glide but I still can't build

```
# github.com/jgensler8/terraform-provider-minikube/minikube
minikube/resource_minikube.go:368:12: undefined: cluster.MachineConfig
minikube/resource_minikube.go:426:22: undefined: bootstrapper.KubernetesConfig
minikube/resource_minikube.go:447:19: undefined: cluster.Config
minikube/resource_minikube.go:627:42: undefined: cluster.Config
minikube/resource_minikube.go:628:9: undefined: cluster.Config
minikube/resource_minikube.go:649:31: undefined: cluster.Config
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ deploy:
   api_key:
     secure: HNzRiN6I8lLNIJy1WDtRqebusIi0zUONyoUQIIyPQJ0xQB4NkJ5HFdU0o1YIOsfPg+FaARTcz+H27HKtAtRtlbD1B1syj4LdGoQ0GcR0Ew0GBLKd/LtxcnDXR4/3/iVZ9C3rJ4YosOSL9pCm96P19AqVaEIUHPqRUOJznlvoznRFbzpJGuV9uROsRGee7heVuttOasEDFF3I3Z2ZqiY6f2Fg/1PXu0eNUlY+FEE2KTNOpwSCXAIAkjBB5sfmU4hvl2W6ZSaPweDJ+AxIiqDqb0hL08/5ucUNfoRl8QAp+Wyg3MHV5QJDDHLln6bJ76HzjT6/dWdOzZnaKqGkKAmfumgG0+UvNiZztN0sa9QMYpNi72LsZ8LTCbVy1jTBq+yCltNmJH/Hcsxhgnp2rWWN1Vji+38kmwLhmycppHelccKNkvSTrxuQaHLO5MIpwJW3zl5mIl+qZw6kol//pDPjDclDZQuXrcNdfRkWpoNpcVgd7fxApA+7omjWdwImdJDzG4iSQbZzy24nYIwMuJJdUEP1s3Go6IZeQPrNgHALiL13TJfKlLW70oQC0CAvz2DLjG5wThoutEQCvTMYxkq11eLeQ0entKhQ54APLisHz9Ks7VdeNN1tTHesVqwanM8lXhKJe0s3br04bQDl/37K5DkCb8agf81pgn80o+xVzp4=
   file_glob: true
-  file: "*.zip"
+  file: releases/*
+  skip_cleanup: true
   on:
-    repo: jgensler8/terraform-provider-minikube
-    branch: master
+    tags: true

--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,11 @@
 RELEASE_VERSION=v0.4.0
+RELEASE_DIR=releases
 GOARCH?=amd64
 BINARY=terraform-provider-minikube_${RELEASE_VERSION}_x4
 LINUX_RELEASE=terraform-provider-minikube_${RELEASE_VERSION}_linux_${GOARCH}
 MAC_RELEASE=terraform-provider-minikube_${RELEASE_VERSION}_darwin_${GOARCH}
 
-default: deps assets_hack linux mac
+default: deps assets_hack linux mac stage
 
 deps:
 	go get -d -v ./...
@@ -13,6 +14,7 @@ assets_hack:
 	make -C ${GOPATH}/src/k8s.io/minikube pkg/minikube/assets/assets.go
 
 clean:
+	rm -rf "${RELEASE_DIR}"
 	rm ${LINUX_RELEASE} ${MAC_RELEASE} ${LINUX_RELEASE}.zip ${MAC_RELEASE}.zip
 
 linux:
@@ -24,3 +26,8 @@ mac:
 	GOOS=darwin GOARCH=${GOARCH} go build -o "${BINARY}"
 	zip "${MAC_RELEASE}.zip" "${BINARY}"
 	rm "${BINARY}"
+
+stage:
+	mkdir "${RELEASE_DIR}"
+	mv "${LINUX_RELEASE}.zip" "${RELEASE_DIR}"
+	mv "${MAC_RELEASE}.zip" "${RELEASE_DIR}"

